INSTALLATION.md

```diff
@@ -152,12 +161,11 @@ Using Mongosh:          1.6.2
 
 For mongosh info see: https://docs.mongodb.com/mongodb-shell/
 
-...
-...
```

Comment:
Please restore. The ellipses indicate that there is other irrelevant content to follow